docs/components/slideout.md

```diff
@@ -80,3 +80,9 @@ There is one default slot which is used to place content into the slideout panel
 ## Events
 
 - `@close` - Emitted when the close button is clicked, anything outside the panel is clicked, or the `esc` key is pressed.
+
+<style lang="scss" scoped>
+.vertical-spacing {
```

Comment:
Does this look like a utility class? 
@portikM should this be renamed? what do you think?

Reply:
No this is good, we are starting to establish `vertical/horizontal-spacing` as a pattern in docs